packages/search-adapter/package.json

```diff
@@ -1,10 +1,32 @@
 {
   "name": "@empathyco/x-adapter",
```

Comment:
Just a question of other package. Why did we rename search-types to x-types but we didn't rename the folder? It can be confusing to find search-types as a folder but having x-types as the package name.

Reply:
Honestly I don't know, but the same happens for every other project here